### ISSUE TITLE
Blog API: Should mention of portfolio be removed

### ISSUE THREAD

**Issue by damon314159** [TheOdinProject/curriculum#28726]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the introduction to this project it mentions that "if you already have a portfolio site" then you can reuse/edit that. I wonder if this is a holdout from the now-rebranded portfolio/homepage project. I wonder if the introduction to this project should be edited to say homepage, or perhaps just have that sentence removed

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-nodejs-blog-api

**Comment by MaoShizhong** [comment]
I think that's where the wording originally came from and definitely don't think it's relevant anymore.

## Acceptance criteria

- [ ] Remove the sentence "If you already have a portfolio site and you want to add your blog posts to that site feel free to do that instead of creating a new site just for that." from the second paragraph of the Node Blog API project.

If you'd like to be assigned to this issue, please comment below. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by lukegrob** [comment]
I'd love to try this, if possible!

**Comment by MaoShizhong** [comment]
All yours then! Let us know here if you encounter any problems.
